client/src/Overview/index.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import ProductInfo from './productInfo.jsx';
+
+// const GH_TOKEN = require('../../../tokens.js');
+const GH_TOKEN = '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';
```

Comment:
@xanzadu Can you re-submit without your GH key? You'll also want to create a new one since this one's out in the wild now.

Reply:
Should be fixed now.